### DISCUSSION TITLE
Possible for Power Apps to update Budibase database?

### DISCUSSION THREAD

**Discussion by ctam19** [Budibase/budibase#15010] [Help]
Hi everyone,

I am using Power Apps to upload my receipts and its records. Is it possible for Power Apps (flow) to update Budibase Database? If anyone has done it, can you show me how?

Thanks!

**Comment by ConorWebb96** [comment]
I'm not aware of any native connection with Power Apps. If you can send the information to APIs, there are a few ways to link Power Apps to Budibase.

You could use a webhook automation in Budibase. When a certain action is performed in Power Apps, send the information to a Budibase automation, where you can create steps that do whatever you wish with that information. It could be as simple as creating a row or something more complex, notifying users and progressing a workflow.
Depending on your needs, you could hook into Budibase's public API and directly create/update/delete rows in tables within Budibase.

I hope this helps.

**Comment by ctam19** [comment]
Hi @ConorWebb96 , Thanks for the reply. 

I figured this out myself. Power Apps Flow can send data via API. So I use POST API from Budibase to send the data to my Budibase DB.
